test/helper/Puppeteer_test.js

```diff
@@ -276,6 +277,25 @@ describe('Puppeteer', function () {
       .then(() => I.seeNumberOfElements('#area1', 1)));
   });
 
+  describe('#startMocking, #mock', () => {
```

Comment:
This test doesn't work, as axios is useless here.
We mock only requests inside browser, not inside NodeJS.

Reply:
Changed to acceptance test.